src/clientSideScene/setup.tsx

```diff
@@ -490,14 +491,15 @@ class SetupSingleton {
             this.camera.updateProjectionMatrix()
             this.dollyZoom(currentFov)
             requestAnimationFrame(animateFovChange) // Continue the animation
+          } else if (frameWaitOnFinish > 0) {
+            frameWaitOnFinish--
+            requestAnimationFrame(animateFovChange) // Continue the animation
           } else {
-            setTimeout(() => {
-              // Once the target FOV is reached, switch to the orthographic camera
-              // Needs to wait a couple frames after the FOV animation is complete
-              this.useOrthographicCamera()
-              this.isFovAnimationInProgress = false
-              resolve(true)
-            }, 100)
+            // Once the target FOV is reached, switch to the orthographic camera
+            // Needs to wait a couple frames after the FOV animation is complete
+            this.useOrthographicCamera()
+            this.isFovAnimationInProgress = false
+            resolve(true)
```

Comment:
Oh just realised I committed with here, 

Not really related to the rest, but not worth moving now.

Waiting for a few frames explicitly is the right way of doing this delay, instead of setTimeout.